ui/components/forms/flavors/select/index.scss

```diff
@@ -22,3 +22,35 @@
     }
   }
 }
+
+#{$css-prefix}form-element__select-container {
+    position: relative;
+    .#{$css-prefix}select {
+        -moz-appearance: none;
+        -webkit-appearance: none;
+        appearance: none;
```

Comment:
you can remove the prefix of `-moz` and `-webkit-`, we have an autoprefixer that handles this